recipes/csbdeep/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "csbdeep" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/csbdeep-{{ version }}.tar.gz
+  sha256: be886751dde0d9e68dab49f2cc149f9bb38b718d6660c31d30722adbf7601327
+
+build:
+  entry_points:
+    - care_predict = csbdeep.scripts.care_predict:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - h5py >=3  # [py>=39]
+    - h5py <3   # [py<39]
+    - tensorflow >=2 
+    - matplotlib-base
+    - numpy
+    - python
+    - scipy
+    - six
+    - tifffile
+    - tqdm
+  run_constrained:
+    - keras >=2.1.2,<2.4
+
+test:
+  imports:
+    - csbdeep
+  commands:
+    - pip list
+    # - pip check
+    - care_predict --help
+  requires:
+    - pip
```

Comment:
We do have a `pytest`-based test suite. Should this be run here?

Reply:
That's usually left to the upstream maintainers, we only need to ensure that the package can be imported and the entry points work. We assume a final release is fully working once those minimum requirements are met. That said, maybe you want to add more import checks?

Reply:
Let's leave it for now, but good to know. Thanks!